applications/tari_console_wallet/src/ui/components/menu.rs

```diff
@@ -0,0 +1,65 @@
+use crate::ui::{components::Component, state::AppState};
+use tui::{
+    backend::Backend,
+    layout::{Constraint, Direction, Layout, Rect},
+    style::{Color, Modifier, Style},
+    text::{Span, Spans},
+    widgets::{Block, Paragraph},
+    Frame,
+};
+
+pub struct Menu {}
+
+impl Menu {
+    pub fn new() -> Self {
+        Self {}
+    }
+}
+
+impl<B: Backend> Component<B> for Menu {
+    fn draw(&mut self, f: &mut Frame<B>, area: Rect, _app_state: &AppState)
+    where B: Backend {
+        let columns = Layout::default()
+            .direction(Direction::Horizontal)
+            .constraints([Constraint::Ratio(4, 5), Constraint::Ratio(1, 5)].as_ref())
+            .split(area);
+
+        let others = Spans::from(vec![
+            Span::styled("LeftArrow", Style::default().fg(Color::Green)),
+            Span::styled(":", Style::default().fg(Color::White)),
+            Span::styled(
+                "PrevTab ",
+                Style::default()
+                    .fg(Color::Magenta)
+                    .bg(Color::LightGreen)
+                    .add_modifier(Modifier::BOLD),
+            ),
+            Span::raw(" "),
+            Span::styled("Tab/RightArrow", Style::default().fg(Color::Green)),
+            Span::styled(":", Style::default().fg(Color::White)),
+            Span::styled(
+                "NextTab ",
+                Style::default()
+                    .fg(Color::Magenta)
+                    .bg(Color::LightGreen)
+                    .add_modifier(Modifier::BOLD),
+            ),
+        ]);
+        let quit = Spans::from(vec![
+            Span::styled("F10/Ctrl-Q/Ctrl-C", Style::default().fg(Color::Green)),
```

Comment:
```suggestion
            Span::styled("F10/Ctrl-Q ", Style::default().fg(Color::Green)),
```

Reply:
Please elaborate on why `Ctrl-C` should not be included/disclosed?

Reply:
Just personal preference really, I think 2 options is enough but all good

Reply:
I think what @delta1 is saying here is that it is reasonable to assume that the user would think ctrl-c would terminate the application without it needing to be explicitly mentioned.

Either way should be fine though.